src/LaunchDarkly.Client/Properties/AssemblyInfo.cs

```diff
@@ -32,5 +32,6 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.2.6.0")]
-[assembly: AssemblyFileVersion("0.2.6.0")]
+[assembly: AssemblyInformationalVersion("0.3.3.0-alpha")]
```

Comment:
This line allows us to push prerelease versions to nuget.org so we can test them out.